src/main.ts

```diff
@@ -18,6 +18,7 @@ import { router } from '@/plugins/router';
  */
 import '@/assets/styles/global.css';
 import '@fontsource/inter';
+import 'uno.css';
```

Comment:
```suggestion
import 'virtual:uno.css'
```

Se me olvidó completamente al iniciar el frontend, mis disculpas. [Fuente](https://unocss.dev/integrations/vite#installation)